scripts/e2e_test_docker.sh

```diff
@@ -39,36 +39,44 @@ BIN_FOLDER=$REPO_ROOT/bin
 TEST_REGEX="${1:-TestDockerKubernetes130SimpleFlow}"
 BRANCH_NAME="${2:-main}"
 
+export AWS_SDK_LOAD_CONFIG=true
+export AWS_CONFIG_FILE=$(pwd)/config_file
+export AWS_PROFILE=e2e-docker-test
 
-cat << EOF > config_file
+cat << EOF > ${AWS_CONFIG_FILE}
 [default]
 output=json
 region=${AWS_REGION:-${AWS_DEFAULT_REGION:-us-west-2}}
 role_arn=$AWS_ROLE_ARN
 web_identity_token_file=/var/run/secrets/eks.amazonaws.com/serviceaccount/token
-[profile e2e-docker-test]
+[profile ${AWS_PROFILE}]
 role_arn=$TEST_ROLE_ARN
 region=${AWS_REGION:-${AWS_DEFAULT_REGION:-us-west-2}}
 source_profile=default
 EOF
 
+unset AWS_ROLE_ARN AWS_WEB_IDENTITY_TOKEN_FILE
+
 INTEGRATION_TEST_INFRA_CONFIG="/tmp/test-infra.yml"
 export T_TINKERBELL_S3_INVENTORY_CSV_KEY="inventory/den80/den80-hardware.csv"
 
+INTEGRATION_TEST_AMI_ID=$(aws ec2 describe-images \
+  --profile ${AWS_PROFILE} \
+  --owners 857151390494 \
```

Comment:
can't account be dynamic?

Reply:
Since we only build it in the build account and share it to these accounts, we are hardcoding the owner here.